### PR TITLE
Updated WAF Limitations & S3 Storage Classes

### PR DESCRIPTION
**Pull request by dshah22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -720,6 +720,8 @@ S3
...
 	-	🔸[Reduced Redundancy Storage (RRS)](https://aws.amazon.com/s3/reduced-redundancy/) has been [effectively deprecated](https://www.quinnadvisory.com/blog/2017/4/13/reduced-redundancy-s3-is-dead), and has lower durability (99.99%, so just four nines) than standard S3. Note that it no longer participates in S3 price reductions, so it offers worse redundancy for more money than standard S3. As a result, there's no reason to use it.
 	-	[Infrequent Access (IA)](https://aws.amazon.com/s3/storage-classes/#Infrequent_Access) lets you get cheaper storage in exchange for more expensive access. This is great for archives like logs you already processed, but might want to look at later. To get an idea of the cost savings when using Infrequent Access (IA), you can use this [S3 Infrequent Access Calculator](http://www.gulamshakir.com/apps/s3calc/index.html).
+	-	[S3 - Intellignet Tiering](https://aws.amazon.com/about-aws/whats-new/2018/11/s3-intelligent-tiering/) storage class is designed to optimize costs by automatically moving data to the most cost-effective access tier, without performance impact or operational overhead.
```
Comment (QuinnyPig) [inline README.md]: "Intelligent" is misspelled in the head.

File: README.md [hunk 2]
```diff
@@ -720,6 +720,8 @@ S3
...
+	-	[S3 - One Zone - IA](https://aws.amazon.com/s3/storage-classes/#__) is for data that is accessed less frequently, but requires rapid access when needed. Unlike other S3 Storage Classes which store data in a minimum of three Availability Zones (AZs), S3 One Zone-IA stores data in a single AZ and costs 20% less than S3 Standard-IA.
```
Comment (QuinnyPig) [inline README.md]: What's that anchor in the link going to?

Comment (dshah22) [reply README.md]: Anchor is going to One-Zone-IA storage class in the aws doc. I know its strange how aws has that anchor too

**Comment by dshah22** [comment]
@Ishaan28malik thanks for approving changes. Can you please also Approve the PR and merge?

Thanks

**Comment by Ishaan28malik** [comment]
@QuinnyPig please merge this.

**Comment by QuinnyPig** [comment]
Excellent work. Merging!

**Pull request merged**